src/main/java/com/gradle/enterprise/conventions/DevelocityConventionsPlugin.java

```diff
@@ -1,17 +1,15 @@
 package com.gradle.enterprise.conventions;
```

Comment:
🤔 I don't think the package `com.gradle.enterprise` is great. I don't think it should be `com.gradle` at all since it is for the open source parts of Gradle. Let's rename those things in a separate PR.